docs/topics/proc_using-openrewrite-recipes.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * docs/cli-guide/master.adoc
+
+
+link:https://docs.openrewrite.org/[OpenRewrite], which is integrated into the {ProductShortName} CLI, uses recipes to automate large-scale, distributed source code refactoring.
```

Comment:
I don't think you need " which is integrated into the {ProductShortName} CLI". If this module is is part of the CLI guide, then the user will figure out fairly quickly that it is integrated into the CLI.

I think you can combine the two sentences. "You can apply OR recipes<link> with the MTA CLI to refactor the source code of Java applications."

Normally we are supposed to say "by using" instead of "with" but it might sound too cumbersome. You decide.